emmet-core/emmet/core/utils.py

```diff
@@ -66,6 +67,57 @@ def _get_sg(struc):
             yield group
 
 
+def form_env(mol_lot: Tuple[Molecule, str]) -> str:
+    """
+    Get the alphabetical formula and solvent environment of a calculation
+    as a string
+
+    :param mol_lot: tuple (Molecule, LevelOfTheory)
+
+    :returns key: str
+    """
+
+    molecule, lot = mol_lot
+    lot_comp = lot.split("/")
+    if lot_comp[2].upper == "VACUUM":
+        env = "VACUUM"
+    else:
+        env = lot_comp[2].split("(")[1].replace(")", "")
+
+    key = molecule.composition.alphabetical_formula
+    key += " " + env
+    return key
+
+
+def group_molecules(
+        molecules: List[Molecule],
+        lots: List[str]
+):
+    """
+    Groups molecules according to composition, charge, environment, and equality
+    
+    Args:
+        molecules (List[Molecule])
+        lots (List[str]): string representations of Q-Chem levels of theory
```

Comment:
Could you list an example of what a "string representation of Q-Chem levels of theory" looks like?